omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -117,6 +117,17 @@
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
 ####
+# External RabbitMQ
+####
+default['private_chef']['external-rabbitmq']['enable'] = false
+default['private_chef']['external-rabbitmq']['actions_user'] = 'actions'
+default['private_chef']['external-rabbitmq']['actions_password'] = 'changeme'
+default['private_chef']['external-rabbitmq']['actions_vhost'] = '/analytics'
```

Comment:
it does not cause erchef requests to fail, but it fails to send messages to analytics. And that can cause the erchef logs to fillup and / or slow response times.

We will provide the users with a script to create the users, vhosts and grant appropriate permissions - although this might not be the best solution.